contracts/0.8.25/vaults/Permissions.sol

```diff
@@ -93,74 +99,161 @@ abstract contract Permissions is AccessControlVoteable {
+     */
     function _withdraw(address _recipient, uint256 _ether) internal virtual onlyRole(WITHDRAW_ROLE) {
-        _unsafeWithdraw(_recipient, _ether);
+        stakingVault().withdraw(_recipient, _ether);
     }
 
+    /**
+     * @dev Checks the MINT_ROLE and mints shares backed by the StakingVault.
+     * @param _recipient The address to mint the shares to.
+     * @param _shares The amount of shares to mint.
+     * @dev The zero checks for parameters are performed in the VaultHub contract.
+     */
     function _mintShares(address _recipient, uint256 _shares) internal onlyRole(MINT_ROLE) {
         vaultHub.mintSharesBackedByVault(address(stakingVault()), _recipient, _shares);
     }
 
+    /**
+     * @dev Checks the BURN_ROLE and burns shares backed by the StakingVault.
+     * @param _shares The amount of shares to burn.
+     * @dev The zero check for parameters is performed in the VaultHub contract.
+     */
     function _burnShares(uint256 _shares) internal onlyRole(BURN_ROLE) {
         vaultHub.burnSharesBackedByVault(address(stakingVault()), _shares);
     }
 
+    /**
+     * @dev Checks the REBALANCE_ROLE and rebalances the StakingVault.
+     * @param _ether The amount of ether to rebalance the StakingVault with.
+     * @dev The zero check for parameters is performed in the StakingVault contract.
+     */
     function _rebalanceVault(uint256 _ether) internal onlyRole(REBALANCE_ROLE) {
         stakingVault().rebalance(_ether);
     }
 
+    /**
+     * @dev Checks the PAUSE_BEACON_CHAIN_DEPOSITS_ROLE and pauses beacon chain deposits on the StakingVault.
+     */
     function _pauseBeaconChainDeposits() internal onlyRole(PAUSE_BEACON_CHAIN_DEPOSITS_ROLE) {
         stakingVault().pauseBeaconChainDeposits();
     }
 
+    /**
+     * @dev Checks the RESUME_BEACON_CHAIN_DEPOSITS_ROLE and resumes beacon chain deposits on the StakingVault.
+     */
     function _resumeBeaconChainDeposits() internal onlyRole(RESUME_BEACON_CHAIN_DEPOSITS_ROLE) {
         stakingVault().resumeBeaconChainDeposits();
     }
 
+    /**
+     * @dev Checks the REQUEST_VALIDATOR_EXIT_ROLE and requests validator exit on the StakingVault.
+     * @param _pubkey The public key of the validator to request exit for.
+     */
     function _requestValidatorExit(bytes calldata _pubkey) internal onlyRole(REQUEST_VALIDATOR_EXIT_ROLE) {
         stakingVault().requestValidatorExit(_pubkey);
     }
 
+    /**
+     * @dev Checks the VOLUNTARY_DISCONNECT_ROLE and voluntarily disconnects the StakingVault.
+     */
     function _voluntaryDisconnect() internal onlyRole(VOLUNTARY_DISCONNECT_ROLE) {
         vaultHub.voluntaryDisconnect(address(stakingVault()));
     }
 
-    function _transferStakingVaultOwnership(address _newOwner) internal onlyIfVotedBy(_votingCommittee()) {
+    /**
+     * @dev Checks the DEFAULT_ADMIN_ROLE and transfers the StakingVault ownership.
```

Comment:
the comment is misleading a bit: we check the confirmation roles array, not the `DEFAULT_ADMIN_ROLE` per se